src/pages/shoppingcart.js

```diff
@@ -0,0 +1,20 @@
+import Link from 'next/link';
+
+import SVG from '../../public/SVG/svg';
+import Header from '../components/Header';
+import ProductListCart from '../components/ProductList/cart';
+
+export default function ShoppingCartPage() {
+	return (
+		<>
+			<Header />
+
+			<Link href="/">
+				<a>
+					<SVG size="40px" color="black" variant="backArrow" />
+				</a>
+			</Link>
```

Comment:
Same here about the `aria-label`. Also, the formatting on this section is a bit off. Maybe check and run prettier again. :)